sqlx-core/Cargo.toml

```diff
@@ -150,6 +150,7 @@ libsqlite3-sys = { version = "0.24.1", optional = true, default-features = false
     "pkg-config",
     "vcpkg",
     "bundled",
+    "bundled-sqlcipher",
```

Comment:
In order to have encryption functionality in my tests I had to add this feature. I am not quite sure I am allowed to do that. But on the other hand, since we use bundled version of `libsqlite3-sys`, why not put into the bundle also `sqlcipher`?

Reply:
If instead you add the following to `[dev-dependencies]` in the root `Cargo.toml`:

```rust
# Needed to test SQLCipher
libsqlite3-sys = { version = "*", default-features = false, features = ["bundled-sqlcipher"] }
```

It will enable that feature just for tests and won't force it on for all users.

Reply:
Done :) Added another test 'it_encrypts' that checks if database encryption actually happens. If someone will remove 'bundled-sqlcipher' from dev-dependencies, it will fail.